contracts/dk/DuelistStakingEarnBox.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.4 <0.9.0;
+pragma abicoder v2;
+
+import "../dao/ERC20.sol";
+
+contract StakingEarnBoxDKT {
+
+/**
+ * numberOfLockDays is a number of days that
+ * user must be stack before unstacking without penalty
+ */
+
+  struct StakingCampaign {
+    uint256 startDate;
+    uint256 endDate;
+    uint256 returnRate;
+    uint256 maxAmountOfToken;
+    uint256 stakedAmountOfToken;
+    uint256 limitStakingAmountForUser;
+    address tokenAddress;
```

Comment:
We need to make sure `tokenAddress` is a smart contract.

Reply:
I will convert the type of `tokenAddress` to `ERC20`.